app/controllers/admin/resources_controller.rb

```diff
@@ -48,5 +56,24 @@ def destroy
     def resource_params
       params.require(:resource).permit(:name, :description, :quantity, :used, :conference_id)
     end
+
+    def check_successful_update(increment_used_resource_flag)
+      if increment_used_resource_flag.present?
+        # Increment/Decrement of Resource via Index Page
+        @resource.used = params[:increment_used_resource_flag].to_i == 1 ? (@resource.used + 1) : (@resource.used - 1)
+        @resource.save
+      else
+        # Update via Edit Page
+        @resource.update_attributes(resource_params)
+      end
+    end
+
+    def get_successful_update_flash(increment_used_resource_flag)
```

Comment:
Is [this](https://github.com/openSUSE/osem/pull/1301/commits/29eedd92cc5d2484d551a2625cd797961cff8664) what you want instead of function??